CONTRIBUTING.md

```diff
@@ -1,11 +1,15 @@
-By submitting a pull request, you represent that you have the right to license
-your contribution to Apple and the community, and agree by submitting the patch
+By submitting a pull request here, you represent that you have the right to license
+your contribution to Apple and the community, and consent by submitting the patch
 that your contributions are licensed under the [Swift
 license](https://swift.org/LICENSE.txt).
 
 ---
 
-Before submitting the pull request, please make sure you have [tested your
+Before submitting a pull request, please make sure you have [tested your
 changes](https://github.com/apple/swift/blob/main/docs/ContinuousIntegration.md)
-and that they follow the Swift project [guidelines for contributing
+and that they follow the Swift project guidelines for [contributing
 code](https://swift.org/contributing/#contributing-code).
+
+---
+
+Thank you for taking your interest for contributing in Swift!
```

Comment:
```suggestion
Thank you for taking interest in contributing to Swift!
```

What do you think about starting, rather than ending, the page with this sentence?